Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Build and test stage
+FROM python:3.8-buster as build
+
+# Upgrade pip
+RUN pip install -U pip
+
+WORKDIR /app
+
+COPY . /app
+
+#install dependencies
+RUN pip install -r requirements.txt
+
+#run unit tests
+RUN python3 -m unittest tests.arcfeed_cms_filtertest
```

Comment:
Let's use pytest instead since it is more popular in python circles

Reply:
done